modules/apps/27-interchain-accounts/keeper/relay_test.go

```diff
@@ -91,3 +100,128 @@ func (suite *KeeperTestSuite) TestTrySendTx() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestOnRecvPacket() {
+	var (
+		path       *ibctesting.Path
+		msg        sdk.Msg
+		txBytes    []byte
+		packetData []byte
+		sourcePort string
+	)
+
+	testCases := []struct {
+		msg      string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"Interchain account successfully executes banktypes.MsgSend", func() {
```

Comment:
@colin-axner I think we should test for multiple message types here in the future. At least the ones that we know are required for our initial use cases i.e. staking. 

I'll do this in a follow-up PR (I need to look into staking a bit). 

wdyt?

Reply:
Sounds good to me, maybe we can try send an ibc-transfer?